### PR TITLE
Usage statement and "header"

### PR DESCRIPTION
**Pull request by Jimmy-ahlberg**
Added some proposed language for us to discuss.

### CODE REVIEW

**Comment by Jimmy-ahlberg** [comment]
This also leaves open the discussion on if we see a value to investigate possible trademarks for the SBOM specification.

File: OpenChain Telco SBOM Specification.md [hunk 1]
```diff
@@ -115,4 +115,6 @@ SBOMs may be subject to confidentiality agreements. A conformant SBOM must, howe
 Some open source software licenses enable any recipient to redistribute the software. In these situations, the recipients should be also able to redistribute the relevant parts of the SBOMs.
 
 ## 4.	Conformant notice: 
-To indicate that the software has a conformant SBOM available, you may use the following statement: “Lorem IPSUM"
+To indicate that the software has a conformant SBOM available, you may use the following statement: "This software is supplied with a SBOM conformant to the OpenChain Telco SBOM Specification v1.0, The specification is available at [Inser homepage]"
```
Comment (CsatariGergely) [inline OpenChain Telco SBOM Specification.md:118]:
```suggestion
To indicate that the software has a conformant SBOM available, you may use the following statement: "This software is supplied with a SBOM conformant to the OpenChain Telco SBOM Specification v1.0, The specification is available at [Insert homepage]"
```

File: OpenChain Telco SBOM Specification.md [hunk 2]
```diff
@@ -115,4 +115,6 @@ SBOMs may be subject to confidentiality agreements. A conformant SBOM must, howe
...
+
+You may at your choosing use the following statement in you Telco Speficatation conformant SBOM “This SBOM conforms to the OpenChain Telco SBOM specification v1.0 [Inser homepage], it is provided to the recipient free of charge, and the recipient is free to redistribute this SBOM to any 3rd that they distribute the corresponding software to, provided that they have all the necessary right to distribute the software to such 3rd party"
```
Comment (CsatariGergely) [inline OpenChain Telco SBOM Specification.md:120]:
```suggestion
You may at your choosing use the following statement in you Telco Speficatation conformant SBOM “This SBOM conforms to the OpenChain Telco SBOM specification v1.0 [Insert homepage], it is provided to the recipient free of charge, and the recipient is free to redistribute this SBOM to any 3rd that they distribute the corresponding software to, provided that they have all the necessary right to distribute the software to such 3rd party"
```

**Review by CsatariGergely — CHANGES_REQUESTED** [review]

File: OpenChain Telco SBOM Specification.md (hunk 1 above)
Comment (winterrocks) [reply OpenChain Telco SBOM Specification.md:118]: Just a typo fix.

File: OpenChain Telco SBOM Specification.md (hunk 2 above)
Comment (winterrocks) [reply OpenChain Telco SBOM Specification.md:120]: Typo fixes.

**Pull request merged**